phpunit.xml.dist

```diff
@@ -8,6 +8,9 @@
   <filter>
     <whitelist>
       <directory suffix=".php">src</directory>
+      <exclude>
+        <directory suffix=".php">src/Logging/V2</directory>
```

Comment:
I played around with the globs a bit, and this seems to work. It should be forward-compatible with newer API versions and potential subfolders beginning with "V". It should match any folder beginning with "V", immediately followed by a non-apha character.

````xml
  <filter>
    <whitelist>
      <directory suffix=".php">src</directory>
      <exclude>
        <directory suffix=".php">src/*/V[!a-zA-Z]</directory>
      </exclude>
    </whitelist>
  </filter>
````